### PR TITLE
add paging to list post and validation form posts

### PR DESCRIPTION
**Pull request by huaphuoc**
Add paging to list post. Add validation form posts: not null, intenger and greater than 0, unique title, min length and max length.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -1,2 +1,8 @@
 class Post < ApplicationRecord
+  validates :title, :content, :quantity,  presence: true
+  validates :title, uniqueness: true
+  validates :title, length: { minimum: 5 }
+  validates :title, length: { maximum: 50 }
+  validates :content, length: { minimum: 10 }
+  validates :quantity, numericality: { only_integer: true, :greater_than => 0}
 end
```
Comment (tran-hai-mulodo) [inline app/models/post.rb]: @huaphuoc 
You can group this validate to clean code as below:
```
validates :title, uniqueness: true, length: { minimum: 5, maximum: 50 }, presence: true
validates :content, length: { minimum: 10 }, presence: true
validates :quantity, numericality: { only_integer: true, :greater_than => 0}, presence: true
```

Comment (huaphuoc) [reply app/models/post.rb]: Ok thanks.

**Comment by huaphuoc** [comment]
I added upload file by paperclip and validation file (file type, file size). Please review it, thanks.

**Comment by tran-hai-mulodo** [comment]
@huaphuoc 
It's OK. I merged. Thanks.

**Pull request merged**